python/src/gen3authz/client/arborist/base.py

```diff
@@ -885,3 +889,20 @@ async def delete_client(self, client_id):
         )
         self.logger.info("deleted client {}".format(client_id))
         return response.code == 204
+
+    async def _create_policy(self, policy_json, skip_if_exists=True):
```

Comment:
can we have `create_policy` call `_create_policy` instead of duplicating the function code?